build-tools/create-packs/ConfigureLocalWorkload.targets

```diff
@@ -24,6 +24,7 @@
       Inputs="$(MSBuildAllProjects);@(_RuntimeListInputs)"
       Outputs="@(_RuntimeListOutputs)">
     <Exec Command="&quot;$(DotNetPreviewTool)&quot; build -t:_GenerateFrameworkList -p:IntermediateOutputPath=%(_RuntimeListOutputs.RelativeDir) -p:AndroidRID=%(_RuntimeListOutputs.AndroidRID) -p:AndroidRuntime=%(_RuntimeListOutputs.AndroidRuntime) -p:Configuration=$(Configuration) &quot;$(MSBuildThisFileDirectory)Microsoft.Android.Runtime.proj&quot;" />
+    <Exec Command="&quot;$(DotNetPreviewTool)&quot; build -t:_GenerateFrameworkList -p:IntermediateOutputPath=$(BuildOutputDirectory)lib\packs\Microsoft.Android.Runtime.$(AndroidApiLevel).android\$(AndroidPackVersion)\data\ -p:Configuration=$(Configuration) &quot;$(MSBuildThisFileDirectory)Microsoft.Android.Runtime.Managed.proj&quot;" />
```

Comment:
When using `<Exec/>`, try to ensure that:

 1. Paths are quoted (in case they have spaces), and
 2. Paths do *not* end in `\`, because if you quote the path this escapes the quote, breaking things.

This may be fine, so long as we somehow ensure/assume that `$(BuildOutputDirectory)` will not contain spaces…

Reply:
I moved this back into the related itemgroup